instrumentation/okhttp/okhttp-3.0/library/src/main/java/io/opentelemetry/instrumentation/okhttp/v3_0/TracingInterceptor.java

```diff
@@ -26,8 +26,11 @@
 
   @Override
   public Response intercept(Chain chain) throws IOException {
-    Context parentContext = Context.current();
     Request request = chain.request();
+    Context parentContext = TracingCallFactory.getCallingContextForRequest(request);
+    if (parentContext == null) {
+      parentContext = Context.current();
```

Comment:
does this seem like the right default, or should we not try to propagate at all if we don't have one in the cache?

Reply:
Let's not propagate at all

Reply:
Ah ok